transform/models/intermediate/vds/int_vds__five_minute_station_agg.sql

```diff
@@ -0,0 +1,58 @@
+{{ config(
+    materialized="incremental",
+    cluster_by=["sample_date"],
+    unique_key=["ID", "SAMPLE_TIMESTAMP"],
+    snowflake_warehouse=get_snowflake_refresh_warehouse()
+) }}
+
+with station_raw as (
+    select
+        *,
+        /* Create a timestamp truncated down to the nearest five
+         minute bucket. This will be the the timestamp on which
+         we aggregate. If a 30-second interval straddles two different
+         buckets, it will be assigned to the one latter one due to
+         the floor() call.
+        */
+        dateadd(
+            'minute',
+            floor(minute(sample_timestamp) / 5) * 5,
+            trunc(sample_timestamp, 'hour')
+        ) as sample_timestamp_trunc
+    from {{ ref('stg_clearinghouse__station_raw') }}
+    {% if is_incremental() %}
+        -- Look back two days to account for any late-arriving data
+        where sample_date > (
+            select dateadd(day, -1, max(sample_date)) from {{ this }}
```

Comment:
```suggestion
            select dateadd(day, -2, max(sample_date)) from {{ this }}
```

if this is indeed meant to be two days